### PR TITLE
Add exception handling

### PR DESCRIPTION
**Pull request by OsPit01**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -19,7 +27,11 @@ public static void main(String[] args) {
                     System.out.println("password");
                     String inputPassword = scanner.nextLine();
                     LoginCommand loginCommand = new LoginCommand();
-                    loginCommand.execute(inputUsername, inputPassword);
+                    try {
+                        loginCommand.execute(inputUsername, inputPassword);
+                    } catch (LoginCommandException e) {
+                        throw new RuntimeException(e);
+                    }
```
Comment (OsPit01) [inline src/Main.java]: there will most likely be an error here)

File: src/exception/LoginCommandException.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package exception;
+
+public class LoginCommandException extends Exception{
```
Comment (VladOsipov) [inline src/exception/LoginCommandException.java]: space between `Exception` and `{`
try CTRL + alt + L

File: src/container/UserContainer.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+import exception.UserNotFoundException;
+import model.User;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserContainer {
+
+    private static final List<User> users = new ArrayList();
```
Comment (VladOsipov) [inline src/container/UserContainer.java]:
```suggestion
    private static final List<User> users = new ArrayList<>();
```

File: src/container/UserContainer.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+    public static final List<User> getUsers() {
```
Comment (VladOsipov) [inline src/container/UserContainer.java]: remove `final` from method

File: src/Main.java (hunk 1 above)
Comment (VladOsipov) [reply src/Main.java]: remove `try-catch` here

**Review by VladOsipov — CHANGES_REQUESTED** [review]

**Review by VladOsipov — APPROVED** [review]

**Pull request merged**